testnet/stacks-node/src/tests/nakamoto_integrations.rs

```diff
@@ -3170,17 +3170,19 @@ fn forked_tenure_is_ignored() {
     );
     let sender_signer_sk = Secp256k1PrivateKey::new();
     let sender_signer_addr = tests::to_addr(&sender_signer_sk);
+    let recipient = PrincipalData::from(StacksAddress::burn_address(false));
     naka_conf.add_initial_balance(
         PrincipalData::from(sender_signer_addr.clone()).to_string(),
         100000,
     );
     let stacker_sk = setup_stacker(&mut naka_conf);
+    let http_origin = format!("http://{}", &naka_conf.node.rpc_bind);
 
     test_observer::spawn();
     let observer_port = test_observer::EVENT_OBSERVER_PORT;
     naka_conf.events_observers.insert(EventObserverConfig {
         endpoint: format!("localhost:{observer_port}"),
-        events_keys: vec![EventKeyType::AnyEvent],
+        events_keys: vec![EventKeyType::AnyEvent, EventKeyType::MinedBlocks],
```

Comment:
Isn't this redundant?

Reply:
No it is not. I know its confusing AF, but EventKeyType::AnyEvent does not subscribe to all events. You must subscribe to them explicitly.

Reply:
Oh, that's fun. Thanks!